api/api.go

```diff
@@ -216,8 +216,6 @@ type CredCreateRequest struct {
 type CredCreateResponse struct {
 	// UUID of the credential that was just created
 	UUID string
-	// CredErr indicates reasonfor failed CredCreate
-	CredErr string
```

Comment:
Removing this from the API? Is this no longer needed? Please get approval from the creator of CredCreateResponse (unless you are it :) ).

Reply:
Yes, not longer needed